native/src/main/java/io/ballerina/stdlib/grpc/Message.java

```diff
@@ -1199,16 +1197,10 @@ public int getSerializedSize() {
                 case DescriptorProtos.FieldDescriptorProto.Type.TYPE_MESSAGE_VALUE: {
                     if (bBMap != null && bBMap.containsKey(bFieldName)) {
                         Object bValue = bBMap.get(bFieldName);
-                        if (bValue instanceof BArray
-                                && fieldDescriptor.getMessageType().getFullName().equals(GOOGLE_PROTOBUF_TIMESTAMP)) {
-                            BArray valueArray = (BArray) bValue;
-                            Message message = new Message(fieldDescriptor.getMessageType(), valueArray);
-                            size += computeMessageSize(fieldDescriptor, message);
-                        } else if (bValue instanceof BArray
-                                && !fieldDescriptor.getMessageType().getFullName().equals(GOOGLE_PROTOBUF_ANY)) {
+                         if (fieldDescriptor.isRepeated() && bValue instanceof BArray) {
```

Comment:
Formatting issue. please check

Reply:
Fixed in https://github.com/ballerina-platform/module-ballerina-grpc/pull/733/commits/ff01dbc5ec7e1f3f6b25a767d2d68a65c93054fd